static/test/javascripts/spec/common/adblock-banner.spec.js

```diff
@@ -0,0 +1,22 @@
+define([
+    'common/modules/commercial/adblock-banner-config',
+    'common/modules/adblock-banner'
+], function (adblockBannerConfig, AdblockBanner) {
+
+    describe('Adblock banner', function () {
+
+        var ukBanners;
+        beforeEach(function () {
+            document.innerHtml = '<div class="top-banner-ad-container--desktop"></div>';
+            ukBanners = adblockBannerConfig.getBanners('UK');
+        });
+
+        it('should be able to render every type of banner', function () {
```

Comment:
So I've split out the synchronous template rendering and am now explicitly testing that - I'm still keen on testing the actual rendering of the actual templates as it can be difficult to see exactly which templates depend on which variables being present